balls.cpp

```diff
@@ -1,7 +1,9 @@
 // BALLS
-#include <iostream>
+#ifndef _BALLS
+#define _BALLS
```

Comment:
Why is there a need?

Reply:
just wanted to make sure we only had one balls

more than that and it might start getting hot in here 😳